tests/common_test.go

```diff
@@ -53,18 +55,18 @@ func expectResponseJSONBodyToBe[T comparable](t *testing.T, res *httptest.Respon
 }
 
 func setTestDB() *gorm.DB {
-	os.Setenv("DEV", "true")
-	os.Setenv("DEV_DB_PATH", "testDB.db")
+	_ = godotenv.Load("../.env")
```

Comment:
Is this required anymore?

Reply:
Yes, we would have to load the env file as gorm needs the env variables.

Reply:
For testing, aren't the env variables directly added to he environment? Or is the `.env` required for local testing?

Reply:
Well I guess it can't hurt so it's fine.

Reply:
Yes, .env file is required for local testing.